app/routes/routes.py

```diff
@@ -113,15 +118,17 @@ def get_question(questionId):
                 return jsonify(temp), 200
         return Response(json.dumps(['Question not Found']),
                         status=404, mimetype='application/json')
-    return jsonify({f'Question {questionId}': 'Has not been added yet'}), 404
+    return jsonify({f'Question {questionId}': 'Has not been added yet'}), 200
```

Comment:
This return message could be improved to "there are no questions or question doesn't exist" since its "if" statement is to query and check if there are any questions in the database.